README.md

```diff
@@ -213,5 +213,5 @@ functions-framework --help
 
 ## For more information
 
-See the examples directory for additional examples, and consult the gem
-documentation on rubydoc.info for full reference documentation.
+See the examples directory for additional examples, and consult
```

Comment:
- Use \` when referring to a folder name
- Use hyperlinks for quick navigation

See [`examples/`](examples/) for additional samples.

Reply:
Done.